### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by kepta**
(closes https://github.com/mapbox/mapbox-sdk-js/issues/278)

@davidtheclark or @danswick for review, please.

### CODE REVIEW

File: services/optimization.js [hunk 1]
```diff
@@ -108,6 +108,12 @@ Optimization.getOptimization = function(config) {
     });
   });
 
+  /**
+   * @typedef {Object} Distribution
+   * @property {number} pickup - indicates the index to the waypoint of the pick-up location
```
Comment (davidtheclark) [inline services/optimization.js]: Let's format these are sentences, with capitalization and punctuation.

What is this referencing the `index` of? Index within what array?

Comment (kepta) [reply services/optimization.js]: Updated the PR, let me know if the wording is better.

File: services/optimization.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ var Optimization = {};
  * @param {Array<OptimizationWaypoint>} config.waypoints - An ordered array of [`OptimizationWaypoint`](#optimizationwaypoint) objects, between 2 and 12 (inclusive).
  * @param {Array<'duration'|'distance'|'speed'>} [config.annotations] - Specify additional metadata that should be returned.
  * @param {'any'|'last'} [config.destination="any"] - Returned route ends at `any` or `last` coordinate.
- * @param {[object, object]} [config.distributions] - Array of objects, each of which includes a `pickup` and `dropoff` property. `pickup` and `dropoff` properties correspond to an index in the coordinates array.
+ * @param {Array<Distribution>} [config.distributions] - An ordered array of  [`Distribution`](#distribution), each of which includes a `pickup` and `dropoff` property. `pickup` and `dropoff` properties correspond to an index in the OptimizationWaypoint array.
```
Comment (davidtheclark) [inline services/optimization.js]: `Distributions` should be plural, I think.

File: services/optimization.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ var Optimization = {};
  * @param {Array<OptimizationWaypoint>} config.waypoints - An ordered array of [`OptimizationWaypoint`](#optimizationwaypoint) objects, between 2 and 12 (inclusive).
  * @param {Array<'duration'|'distance'|'speed'>} [config.annotations] - Specify additional metadata that should be returned.
  * @param {'any'|'last'} [config.destination="any"] - Returned route ends at `any` or `last` coordinate.
- * @param {[object, object]} [config.distributions] - Array of objects, each of which includes a `pickup` and `dropoff` property. `pickup` and `dropoff` properties correspond to an index in the coordinates array.
+ * @param {Array<Distributions>} [config.distributions] - An ordered array of  [`Distributions`](#distributions), each of which includes a `pickup` and `dropoff` property. `pickup` and `dropoff` properties correspond to an index in the OptimizationWaypoint array.
```
[the remark this answers is not in the document]
Comment (kepta) [reply services/optimization.js]: Gotcha, I have updated the commit. Let me know if I get it right.

**Pull request merged**